_rules/SC1-2-video-media-alternative.md

```diff
@@ -0,0 +1,130 @@
+---
+name: Video as a media alternative for text
+description: |
+  This rule checks non-streaming `video` is a media alternative for text on the page.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output](#audio-output)
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video contains audio.
+
+### Expectation 1
+
+All the information contained in each target element is available as text (directly or via text alternatives) that is [visible on the page](#visible-on-the-page) and [exposed to assistive technolgies](#exposed-to-assistive-technologies).
+
+### Expectation 2
+
+Each target element has a label indicating the `video` is an alternative to text on the page.
+
+### Expectation 3
+
+The label (from expectation 2) is [visible on the page](#visible-on-the-page) and [exposed to assistive technolgies](#exposed-to-assistive-technologies)
```

Comment:
"from expectation 2" seems quite procedural. I think expectations 2 and 3 should be combined.

Reply:
Also, "the label" can be confused with <label> elements. Yet I can't think of another term. Maybe add a clarification note?